include/aspect/structured_data.h

```diff
@@ -621,15 +584,6 @@ namespace aspect
         std::vector<std::string>
         get_column_names() const;
 
-        /**
-        * Returns the coordinates at which profile data is stored. This function
-        * can be used to determine the number of data points, or to query
-        * data only at exactly the positions at which it is available (avoiding
-        * interpolation).
-        */
-        const std::vector<double> &
-        get_coordinates() const;
```

Comment:
...whereas this one, in `AsciiDataProfile<dim>` that happens to have the same name, doesn't actually take an argument. The documentation doesn't say so, but it actually returns the interpolation points *in coordinate direction zero*. That's just confusing, and rather than document the issue, I would have renamed the function. But because I see no reason to expose this information to begin with, I think removing it is the best avenue.